Docs/Tools/fetchStats.sh

```diff
@@ -8,7 +8,7 @@ fi
 
 NEXT_URL=$(echo "https://osmcha.org/api/v1/changesets/?date__gte=2021-01-01&date__lte=$DATE&editor=mapcomplete&page=$COUNTER&page_size=1000")
 rm stats.*.json
-while [[ "$NEXT_URL" != "" ]]
+while [[ "$NEXT_URL" != "null" ]]
```

Comment:
on my system that loop didn't terminate properly, but eventually finished once the file could not be found.